docs/source/building-blocks.rst

```diff
@@ -4,6 +4,18 @@ Building Blocks
 Curated Transformers provides building blocks to create your own transformer
 models.
 
+Models
+------
+
+.. autoclass:: curated_transformers.models.transformer.TransformerCausalLM
+   :members:
+
+.. autoclass:: curated_transformers.models.transformer.TransformerDecoder
+   :members:
+
+.. autoclass:: curated_transformers.models.transformer.TransformerEncoder
+   :members:
```

Comment:
🤔 aren't these more like building blocks?

Reply:
I suppose they could be seen that way. On the other hand, `DecoderModule` and co are in the other pages.

Reply:
Let me try if we can have it in two places...

Reply:
That didn't look nice, so I added them to encoders/decoders/causal LMs. But on those pages I made separate sections for `Base Classes`, so that these base classes are clearly separated from the concrete architecture classes.